### PR TITLE
Correction : Fixed array overflow

### PR DESCRIPTION
**Pull request by ayaankhan98**
fixing the array overflow in `sorting/quick_sort.cpp`
related to issue #799

### CODE REVIEW

File: sorting/quick_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (kvedala) [inline sorting/quick_sort.cpp:1]: Please use the Doxygen commuting style.
The comments should start with 
```c
/**
 *
 */
```

File: sorting/quick_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *      copyright The Algorithms
+ *      Author - 
+ *      Correction - ayaankhan98
+ *
+ */
+
+#include <cstdlib>
+#include <iostream>
+
+int partition(int arr[], int low, int high) {
```
Comment (kvedala) [inline sorting/quick_sort.cpp:12]: please add documentation to the code.

Comment (ayaankhan98) [reply sorting/quick_sort.cpp:12]: okay i will fix that in another PR

Comment (ayaankhan98) [reply sorting/quick_sort.cpp:12]: @kvedala new PR #801

Comment (kvedala) [reply sorting/quick_sort.cpp:12]: Thank you. will follow that now 👍

Comment (kvedala) [reply sorting/quick_sort.cpp:12]: You can mark #801 as draft pull-request and mark it "ready for review" when it is. that way, I will be better able to review it.

Comment (ayaankhan98) [reply sorting/quick_sort.cpp:12]: okay sure

**Pull request merged**